packages/app/src/systems/Bridge/components/BridgeTxOverview/BridgeTxOverview.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { cssObj } from '@fuel-ui/css';
+import { Box } from '@fuel-ui/react';
+import { bn } from 'fuels';
+
+import ethImage from '../../../../../public/eth-logo.svg';
```

Comment:
Can we use `~public` to `avoid ../.,......`

Reply:
I'll try your suggestion.  IIRC we need to explicitly import images like this for storybook.

Reply:
your suggestion did not work.  I tried looking in other repos and the storybook docs for an example of how to handle this in stories, but I haven't been able to find any examples.

Reply:
You probably need to configure the alias from tsconfig on the storybook a example can be found on the fuels-wallet repo.